build/configs/imxrt1050-evk/loadable_elf_apps/defconfig

```diff
@@ -859,8 +862,10 @@ CONFIG_C99_BOOL8=y
 # CONFIG_LIBTUV is not set
 # CONFIG_STRESS_TOOL is not set
 # CONFIG_VOICE_SOFTWARE_EPD is not set
-# CONFIG_RTK_WIFI is not set
 # CONFIG_EXTERNAL_VEC is not set
+# CONFIG_LIBLZMA is not set
```

Comment:
why other flags are part of this patch?

Reply:
The new defconfig was created with .config content after make menuconfig command.
This will be the proper setting for defconfig